docs/examples/generatedCanonicalFields.json

```diff
@@ -0,0 +1,822 @@
+    "shortCode": "personal_travel_reimbursement_explanation",
+    "dataType": "string"
+  },
+  {
+    "label": "Geographic Focus",
+    "shortCode": "geo_place_relationships",
+    "dataType": "string"
+  },
+  {
+    "label": "Locations under US sanction",
+    "shortCode": "funds_in_locations_under_us_sanction",
+    "dataType": "string"
+  },
+  {
+    "label": "Locations Under Sanction Explanation",
+    "shortCode": "locations_under_sanction_explanation",
+    "dataType": "string"
+  },
+  {
+    "label": "Will this project support research?",
+    "shortCode": "will_this_project_support_research",
+    "dataType": "string"
+  },
+  {
+    "label": "Involves desk research?",
+    "shortCode": "desk_research",
+    "dataType": "boolean"
+  },
+  {
+    "label": "Collects new data from individuals?",
+    "shortCode": "collect_new_data_from_individuals",
+    "dataType": "boolean"
+  },
+  {
+    "label": "Collects personal data?",
+    "shortCode": "collects_personal_data",
+    "dataType": "boolean"
+  },
+  {
+    "label": "Describe the research activities for collecting personal data",
+    "shortCode": "new_and_personal_data_describe_the_research_activities",
+    "dataType": "string"
+  },
+  {
+    "label": "Will the research be published?",
+    "shortCode": "will_the_research_be_published",
+    "dataType": "boolean"
+  },
+  {
+    "label": "HSR Policy",
+    "shortCode": "hsr_policy",
+    "dataType": "boolean"
+  },
+  {
+    "label": "Research involves minors",
+    "shortCode": "research_involve_minors",
+    "dataType": "boolean"
+  },
+  {
+    "label": "Does your organization have a child protection policy?",
+    "shortCode": "child_protection_policy",
+    "dataType": "boolean"
+  },
+  {
+    "label": "Funds for fiscal agent duties, research or conferences",
+    "shortCode": "non_us_orgs_fiscal_agent_or_research_or_conference",
+    "dataType": "string"
+  },
+  {
+    "label": "Fiscal Agent Duties / Research / Conference Explanation",
+    "shortCode": "fiscal_agent_duties_research_conference_explanation",
+    "dataType": "string"
+  },
+  {
+    "label": "Additional Information from Applicant",
+    "shortCode": "additional_information_requested",
+    "dataType": "string"
+  },
+  {
+    "label": "Latitude",
+    "shortCode": "bing_latitude",
+    "dataType": "number"
+  },
+  {
+    "label": "Longitude",
+    "shortCode": "bing_longitude",
+    "dataType": "number"
+  },
+  {
+    "label": "Neighborhood",
+    "shortCode": "bing_neighborhood",
+    "dataType": "string"
+  },
+  {
+    "label": "Abstract",
+    "shortCode": "abstract_discussion_and_justification",
+    "dataType": "string"
+  },
+  {
+    "label": "Amount recommended",
+    "shortCode": "amount_recommended",
+    "dataType": "number"
+  },
+  {
+    "label": "Purpose Statement",
+    "shortCode": "purpose_statement",
+    "dataType": "string"
+  },
+  {
+    "label": "Org Name",
+    "shortCode": "Org Name",
```

Comment:
Here we see a duplicate field because two spreadsheets were processed having the same core/PDC fields. When using this JSON to post to the PDC service to seed canonical fields, this is no big deal because the first one to be posted wins. These should be identical in all cases except when there are different types determined via the Funder Field Format.

Reply:
This is still kind of an issue where the original spreadsheets are missing the JSON schema type everyone prefers for PDC.